### PR TITLE
UPCommons.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: UPCommons.js [hunk 1]
```diff
@@ -38,30 +38,32 @@
 
 function detectWeb(doc, url) {
 	var type = ZU.xpathText(doc, '//meta[@name="DC.type"]/@content');
-	if (url.indexOf('/handle/')>-1 && type) {
-		if (itemTypes[type]!=null) return itemTypes[type];
+	if (url.includes('/handle/') && type) {
+		if (itemTypes[type] != null) return itemTypes[type]; // eslint-disable-line no-eq-null
```
Comment (zuphilip) [inline UPCommons.js]: Oh, we can do better here. The variable `itemType` is defined below and I guess we just have to make sure that `type` is a key of it.

File: UPCommons.js [hunk 2]
```diff
@@ -36,32 +36,33 @@
...
+
+
 function detectWeb(doc, url) {
 	var type = ZU.xpathText(doc, '//meta[@name="DC.type"]/@content');
-	if (url.indexOf('/handle/')>-1 && type) {
-		if (itemTypes[type]!=null) return itemTypes[type];
-		else return "document";
-	} else if (getSearchResults(doc, true)) {
+	if (url.includes('/handle/') && type) {
+    return itemTypes[type] || "document";
```
Comment (zuphilip) [inline UPCommons.js]: Fix indenting

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**